src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,125 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.UserInvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        User sameUser = new User();
+        sameUser.setLogin("same_login");
+        sameUser.setAge(20);
+        sameUser.setPassword("same_password1234");
+        Storage.PEOPLE.add(sameUser);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.PEOPLE.remove(user);
+    }
+
+    @Test
+    void duplicatesLogin_NotOk() {

```

Comment:
```suggestion
    void register_duplicatesLogin_NotOk() {
```
Better add method name. Because, if you add a new method it will be difficult to understand which method is testing